src/test/scala/mesosphere/marathon/core/launcher/impl/OfferProcessorImplTest.scala

```diff
@@ -104,7 +106,9 @@ class OfferProcessorImplTest extends UnitTest {
     "match successful, launch tasks unsuccessful" in new Fixture {
       Given("an offer")
       val dummySource = new DummySource
-      val tasksWithSource = tasks.map(task => InstanceOpWithSource(dummySource, f.provision(task._1, task._2, task._3)))
+      val tasksWithSource = tasks.map {
+        case (taskInfo, task, instance) => InstanceOpWithSource(dummySource, f.provision(taskInfo, instance.instanceId, instance.agentInfo.get, instance.runSpecVersion, task, clock.now()))
```

Comment:
I think we have a 120 or 140 chars line limit. Also why not
```scala
case (taskInfo, task, Instance(instanceId, Some(agentInfo), runSpecVersion)) => 
  val whateverThisIs =  f.provision(taskInfo, instanceId, agentInfo, runSpecVersion, task, clock.now())
  InstanceOpWithSource(dummySource, whateverThisIs)
```

Reply:
I don't want to be counting chars in my IDE. We should have a formatter than that will enforce it for me...

And re the extractor: I'll use that, thanks...